frontend/simple/js/state.js

```diff
@@ -190,13 +213,13 @@ const actions = {
     {contractId, hash, entry}: {contractId: string, hash: string, entry: Object}
   ) {
     // verify we're expecting to hear from this contract
-    if (!state.pending.includes(contractId) && !state.contracts[contractId]) {
+    if (!state.pending.includes(contractId) && !state.contracts[ contractId ]) {
```

Comment:
To make my life easier as a reviewer, it would be very much appreciated if spacing can be left alone, unless there's a good reason to change it, in which I'd like to know what that reason is *before* changes are made, as I have to check each modified line not just for changes to spacing but anything else.